teamA/jinwoo/controllers/postController.js

```diff
@@ -0,0 +1,51 @@
+const postService = require('../services/postService');
+
+const postUp = async (req, res) => {
```

Comment:
```suggestion
const createPost = async (req, res) => {
```

게시물 생성과 관련 있는 코드 같네요! 그렇다면 createPost 라고 액션과 대상을 묶어서 네이밍해주시면 조금더 일반적인 네이밍 컨벤션에 맞출 수 있습니다!